x-pack/plugins/lists/server/scripts/quick_start_value_list_references.sh

```diff
@@ -0,0 +1,47 @@
+#!/bin/sh
```

Comment:
I get the following when I run this... I think you just need to change it to `/bin/bash` instead of `/bin/sh` ...

![image](https://user-images.githubusercontent.com/611653/102238873-42ca5300-3ec4-11eb-802f-66cfdf31a344.png)

Reply:
Hmmmm, I was following suit of what all the other scripts in: [x-pack/plugins/lists/server/scripts](https://github.com/elastic/kibana/tree/8aaa5f082007aa7227c64c2239f619793d1d37ff/x-pack/plugins/lists/server/scripts).

Looks like it's complaining about line 13 in `post_exception_list.sh` here:

https://github.com/elastic/kibana/blob/ab2600f823f04740ef3c0a9de4312eea5fd84236/x-pack/plugins/lists/server/scripts/post_exception_list.sh#L13

Does changing to `/bin/bash` for that script make it work for you? I haven't had any issues locally (on zsh though), but we should use whatever is most compatible across systems.

Reply:
@spong Running the script as `bash post_exception_list.sh` did work for me, but I didn't try changing the default shell in the file. I don't have the branch pulled currently, but I can try again in a bit. I'd expect it will work though.

Reply:
@spong Just ran a little test locally. The problem is `sh` doesn't support arrays, but `bash` does. So any of these scripts that use the `()` array notation probably should be updated to use `bash` by default.

Reply:
(I realize that's probably out of scope for this PR, but it's something we should address eventually IMO).

Reply:
@madirey -- complete agree! Updated all scripts that reference lists in 5ffb88c 🙂

Reply:
Thanks!